java/src/main/java/com/google/cloud/dataproc/templates/hive/HiveToGCS.java

```diff
@@ -122,4 +94,35 @@ public void runTemplate() {
     LOGGER.info("HiveToGcs job completed.");
     spark.stop();
   }
+
+  void validateInput() {
```

Comment:
Shall we consider moving validateInput to BaseTemplate? I understand that input parameters are different for each parameter but its worth consolidating them in base class via var args or lambada function. We can create a seperate jira for this if required.

Reply:
yeah we should create a separate git issue for code refactoring.